tools/integration_tests/run_e2e_tests.sh

```diff
@@ -139,6 +139,7 @@ function install_packages() {
   sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
   export PATH=$PATH:/usr/local/go/bin
   sudo apt-get install -y python3
+  export PATH=$PATH:/usr/bin/python3
```

Comment:
Is this statement really necessary? I'd expect that after installation, python3 should automatically be available in the path.